test/extensions/filters/http/fault/fault_filter_integration_test.cc

```diff
@@ -149,17 +149,15 @@ TEST_P(FaultIntegrationTestHttp2, ResponseRateLimitTrailersBodyFlushed) {
       codec_client_->makeHeaderOnlyRequest(default_request_headers_);
   waitForNextUpstreamRequest();
   upstream_request_->encodeHeaders(default_response_headers_, false);
-  Buffer::OwnedImpl data(std::string(1152, 'a'));
+  Buffer::OwnedImpl data(std::string(127, 'a'));
   upstream_request_->encodeData(data, false);
-  decoder->waitForBodyData(1024);
 
   // Advance time and wait for a tick worth of data.
```

Comment:
Here time is not being advanced, right? It is just the data that now gets written due to the initial trigger after the bucket is reset. Same below in 185?